### ISSUE TITLE
ValueError: Passing parameters norm and vmin/vmax simultaneously is not supported

### ISSUE THREAD

**Issue by zhoudreames** [deeptools/pyGenomeTracks#438]
My code:
```
pyGenomeTracks --tracks test.ini --region 14:1-25803445 --outFileName test_track.pdf
```
test.ini
```
[x-axis]
where = top

[hic matrix]                                                                                                                                         | 
file = merge.100k_100000.cool
title = Hi-C data
depth = 300000
transform = log1p
file_type = hic_matrix

```
My python is 3.7, but i got the error:
```
INFO:hicmatrix.HiCMatrix:Number of poor regions to remove: 68 {'1': 6, '2': 7, '3': 11, '4': 1, '5': 1, '6': 1, '10': 17, '12': 1, '13': 2, '14': 14, '16': 1, '17': 1, 'X': 1, 'Y': 4}
INFO:hicmatrix.HiCMatrix:found existing 68 nan bins that will be included for masking 
INFO:pygenometracks.tracksClass:time initializing track(s):
INFO:pygenometracks.tracksClass:56.690704584121704
DEBUG:pygenometracks.tracksClass:Figure size in cm is 40 x 1.6555141243936276. Dpi is set to 72

INFO:pygenometracks.tracksClass:plotting 1. [hic matrix]
INFO:pygenometracks.tracks.GenomeTrack:setting min, max values for track 1. [hic matrix] to: 5, 7.0

Traceback (most recent call last):
  File "/home/Mpzhang/miniconda3/envs/hicexplorer/bin/pyGenomeTracks", line 11, in <module>
    main(args)
  File "/home/Mpzhang/miniconda3/envs/hicexplorer/lib/python3.7/site-packages/pygenometracks/plotTracks.py", line 307, in main
    trp.plot(args.outFileName, *region, title=args.title)
  File "/home/Mpzhang/miniconda3/envs/hicexplorer/lib/python3.7/site-packages/pygenometracks/tracksClass.py", line 241, in plot
    track.plot(plot_axis, chrom, start, end)
  File "/home/Mpzhang/miniconda3/envs/hicexplorer/lib/python3.7/site-packages/pygenometracks/tracks/HiCMatrixTrack.py", line 245, in plot
    self.img = self.pcolormesh_45deg(ax, matrix, start_pos, vmax=vmax, vmin=vmin)
  File "/home/Mpzhang/miniconda3/envs/hicexplorer/lib/python3.7/site-packages/pygenometracks/tracks/HiCMatrixTrack.py", line 314, in pcolormesh_45deg
    vmin=vmin, vmax=vmax, cmap=self.cmap, norm=self.norm)
  File "/home/Mpzhang/miniconda3/envs/hicexplorer/lib/python3.7/site-packages/matplotlib/__init__.py", line 1414, in inner
    return func(ax, *map(sanitize_sequence, args), **kwargs)
  File "/home/Mpzhang/miniconda3/envs/hicexplorer/lib/python3.7/site-packages/matplotlib/axes/_axes.py", line 6075, in pcolormesh
    collection._scale_norm(norm, vmin, vmax)
  File "/home/Mpzhang/miniconda3/envs/hicexplorer/lib/python3.7/site-packages/matplotlib/cm.py", line 381, in _scale_norm
    "Passing parameters norm and vmin/vmax simultaneously is "
ValueError: Passing parameters norm and vmin/vmax simultaneously is not supported. Please pass vmin/vmax directly to the norm when creating it.

```

**Comment by zhoudreames** [comment]
@bgruening @Phlya @dpryan79 @fidelram  I am so sorry to disturb you, can you help me ? Thanks~

**Comment by lldelisle** [comment]
Could you list all packages so I can reproduce the bug?
Thanks

**Comment by lldelisle** [comment]
`conda env list`

**Comment by lldelisle** [comment]
I think this has been solved by: https://github.com/deeptools/pyGenomeTracks/pull/305/files#diff-5c05426ae26dc74e1920c66d4f3461c7f2cfeda0733815b443bb6e99191041bd
which has been included in release 3.6. I guess this comes from version of matplotlib > 3.1.1 while version of pyGenomeTracks < 3.6. This should not be possible with a conda installation.

**Comment by lldelisle** [comment]
Feel free to reopen if you still have issues
